### PR TITLE
build: explain the existence of extra-libs.zip

### PR DESCRIPTION
**Pull request by jrudolph**

### CODE REVIEW

File: http-tests/src/test/extra-libs.zip.README [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Archive:  i have späces.jar
+  Length      Date    Time    Name
+---------  ---------- -----   ----
+        0  2015-07-02 15:11   META-INF/
+       68  2015-07-02 15:11   META-INF/MANIFEST.MF
+       19  2015-07-02 15:11   test-resource.txt
+---------                     -------
+       87                     3 files
+```
```
Comment (mdedetrich) [inline http-tests/src/test/extra-libs.zip.README:24]: nit: newline

**Review by mdedetrich — APPROVED** [review]
lgtm

**Comment by pjfanning** [comment]
Are we sure that will keep everyone happy? I have my doubts. Can we consider removing the zip and building it at test run time?

**Comment by jrudolph** [comment]
I would like to avoid building it at build time because then we might not uncover the encoding issues. The whole point is to have an example that we know has been built correctly and then test that this works while testing/building.

**Comment by pjfanning** [comment]
I'm happy to have the file but I'm not the one blocking the release

**Comment by justinmclean** [comment]
You can leave it there but there is a chance some IPMC member might vote -1 as they consider it a binary file in a source release. It doesn't contain compiled code so I think it is OK - but you need to look inside it to find that out.

**Comment by mdedetrich** [comment]
> You can leave it there but there is a chance some IPMC member might vote -1 as they consider it a binary file in a source release. It doesn't contain compiled code so I think it is OK - but you need to look inside it to find that out.

There is no binary code contained in `extra-spaces.zip`. It contains a single file called `i have späces.jar` and that jar itself only contains metadata + a text file, i.e.
```
<@incubator-pekko-http/h/s/test>-<⎇ update-latest-pekko-sbt-paradox>-<*>-> unzip i\ have\ späces.jar 
Archive:  i have späces.jar
   creating: META-INF/
  inflating: META-INF/MANIFEST.MF    
  inflating: test-resource.txt   
  ```

**Pull request merged**